### PR TITLE
New pagination in list-archives

### PR DESCRIPTION
**Pull request by kogierman**
## 🚀 Links

- [Feature branch](https://developers.labs.livechat.com/docs/feature/API-5745-list-archives-pagination)
- [Jira](https://livechatinc.atlassian.net/browse/API-5745)

### CODE REVIEW

File: content/messaging/agent-chat-api/changelog/index.mdx [hunk 1]
```diff
@@ -17,7 +17,8 @@ This document is the record of all the changes in the **Agent Chat API**, Web an
...
 - There's a new push [`chat_unfollowed`](/messaging/agent-chat-api/v3.2/rtm-reference/#chat_unfollowed) complementary to the **Unfollow Chat** method.
 - There's a new `include_chats_without_threads` filter in the  **List Chats** ([Web](/messaging/agent-chat-api/v3.2/#list-chats) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#list-chats)) method.
-- There's a new `agents` filter in the  **List Archives** ([Web](/messaging/agent-chat-api/v3.2/#list-archives) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#list-archives)) method. It replaces the `agents_ids` filter. 
+- There's a new `agents` filter in the  **List Archives** ([Web](/messaging/agent-chat-api/v3.2/#list-archives) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#list-archives)) method. It replaces the `agents_ids` filter.
+- There's a new pagination with hashed page IDs in the  **List Archives** ([Web](/messaging/agent-chat-api/v3.2/#list-archives) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#list-archives)) method. It replaces old pagination.
```
Comment (litwinow) [inline content/messaging/agent-chat-api/changelog/index.mdx:21]: I think it'd be nice to note that the pagination is now uniform across whole API (if it is but I don't recall any method that uses other pagination format).

Comment (ale-kacperczyk) [reply content/messaging/agent-chat-api/changelog/index.mdx:21]: We could try the following (format proposed by @gluszczyk in #656):
```
- the **List Archives** ([Web](/messaging/agent-chat-api/v3.2/#list-archives) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#list-archives)) method:
  - a new `agents` filter replacing the `agents_ids` one
  - new pagination with hashed page IDs replacing the old pagination format. Pagination is now uniform across the whole Agent Chat API.
  -  a new `events.types` filter

Comment (ale-kacperczyk) [reply content/messaging/agent-chat-api/changelog/index.mdx:21]: I think @litwinow is right, get_archives/list_archives was the last one with the old pagination format

File: content/messaging/agent-chat-api/v3.2/index.mdx [hunk 2]
```diff
@@ -1266,11 +1266,17 @@ The list classification is based on threads; 1 chat per 1 thread. Thus, the same
...
-| `pagination.limit`                                    | No       | `number`  | Default: 25, min: 0, max: 100                                                             |
+| `page_id`                                             | No       | `string`  |                                                                                           |
+| `sort_order` **\***                                   | No       | `string`  |  Default: `desc`                                                                          |
+| `limit`                                               | No       | `number`  | Default: 10, min: 1, max: 100                                                             |
 
 **\*)**
+
+`sort_order` can take the following values:
+
+- `asc` - oldest chats first
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/v3.2/index.mdx:1277]:
```suggestion
- `asc` - the oldest chats first
```

File: content/messaging/agent-chat-api/v3.2/index.mdx [hunk 3]
```diff
@@ -1266,11 +1266,17 @@ The list classification is based on threads; 1 chat per 1 thread. Thus, the same
...
+- `desc` - newest chats first
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/v3.2/index.mdx:1278]:
```suggestion
- `desc` - the newest chats first
```

File: content/messaging/agent-chat-api/v3.2/index.mdx [hunk 4]
```diff
@@ -1369,10 +1375,9 @@ curl -X POST \
       "is_followed": false
     }
   ],
-  "pagination": {
-    "page": 1,
-    "total": 1
-  }
+  "found_chats": 42,
+  "next_page_id": "MTUxNzM5ODEzMTQ5Ng==", // optional
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/v3.2/index.mdx:1379]:
```suggestion
  "next_page_id": "MTUxNzM5ODEzMTQ5Ng==",
```

File: content/messaging/agent-chat-api/v3.2/index.mdx [hunk 5]
```diff
@@ -1369,10 +1375,9 @@ curl -X POST \
...
+  "previous_page_id": "MTUxNzM5ODEzMTQ5Ng==" // optional
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/v3.2/index.mdx:1380]:
```suggestion
  "previous_page_id": "MTUxNzM5ODEzMTQ5Ng=="
```

Comment (ale-kacperczyk) [reply content/messaging/agent-chat-api/v3.2/index.mdx:1380]: I suggest we avoid those "JSON comments" when possible. I'll be soon moving all the responses to individual JSON files and will need to delete the "comments" anyway.

File: content/messaging/agent-chat-api/v3.2/index.mdx (hunk 4 above)
Comment (ale-kacperczyk) [reply content/messaging/agent-chat-api/v3.2/index.mdx:1379]: Also in RTM

File: content/messaging/agent-chat-api/v3.2/index.mdx (hunk 3 above)
Comment (ale-kacperczyk) [reply content/messaging/agent-chat-api/v3.2/index.mdx:1278]: Also in RTM

**Pull request merged**